server/src/com/cloud/servlet/ConsoleProxyServlet.java

```diff
@@ -245,8 +247,15 @@ private void handleThumbnailRequest(HttpServletRequest req, HttpServletResponse
         }
     }
 
-    private void handleAccessRequest(HttpServletRequest req, HttpServletResponse resp, long vmId) {
+    private void handleAccessRequest(HttpServletRequest req, HttpServletResponse resp, long vmId,Boolean webSocketRequest) {
```

Comment:
Add a space before `Boolean`, you may use `boolean` to avoid doing `null ` checks everywhere